react-app/src/components/Result.js

```diff
@@ -10,8 +46,12 @@ function Result(props) {
           props.setShowLogs(false);
         }}
       >
-        Patient {props.id}
+        Patient {props.id + 1}
```

Comment:
I have one more question for you and @Dtphelan1. I think previously we have been trying to keep from assuming that the id we display will be an integer, so I'm not sure if we should be incrementing this value here. Was there a reason you decided to change it here? It seems like it might be slightly more complicated to change the id we pass in based on how it gets used in other places, so before we decide to change things, I'm curious if there was a reason for the change and what you and Dylan both think about displaying a `Patient 0` vs `Patient 1` for the first patient.

Reply:
The id that's displayed is the iterator variable from here, so it's guaranteed to be an int:
`const list = props.extractedData.map((bundle, i) => (
    <Result bundle={bundle} id={i} setPatientID={props.setPatientID} key={i} setShowLogs={props.setShowLogs} />
  ));`

In ResultPage, patientID (which is modified whenever someone clicks on a patient) is used as an index variable to determine which JSON schema gets passed to PatientData:
`<PatientData
              patientJson={props.extractedData[patientID]}
              id={patientID}
              showLogs={showLogs}
              loggedMessages={props.loggedMessages}
            /><PatientData
              patientJson={props.extractedData[patientID]}
              id={patientID}
              showLogs={showLogs}
              loggedMessages={props.loggedMessages}
            />`

Since it's generated by calling extractedData.map() used to access bundles in the extractedData array, the count starts at 0. However, I thought starting with 0 in the output might seem strange/confusing to people who don't have a computer science background.

Reply:
The 'Patient X' thing was also originally intended to be a placeholder for some more useful title. If you want, I could use fhirpath to retrieve the patient's name or mrn id, and display that instead.

Reply:
I definitely agree with you on both points - right now the id is guaranteed to be an integer and it would probably be nice to not index a list by zero. However, I think [these comments on previous PRs](https://github.com/mcode/mcode-extraction-ui/pull/3#discussion_r653012115) also discuss how even though we are only using an integer for right now, we want to make sure we aren't locked into using an integer id. This would be another example of when we want to be a little more flexible around how we develop this component to help us ensure we can support different types of data in the future.

I do really like the idea of using `fhirpath` to retrieve the name or another piece of information here. We can discuss at stand up (which is right now haha) about if we want to do that now or if we want to revert this line and do the fhirpath piece separately.

Reply:
Following up on the conversation we had at stand up, we'll plan to change back the display here to just be `props.id` so we avoid requiring an integer, but I'll make a task to explore what might the best way to label each patient in the side bar.